package.json

```diff
@@ -32,7 +32,7 @@
 		"@types/mixpanel-browser": "^2.47.5",
 		"@types/node": "^20.10.4",
 		"@uiw/react-md-editor": "^3.20.5",
-		"axios": "^1.2.1",
+		"axios": "1.7.4",
```

Comment:
I like to keep "^" in the versions, however I already had a problem with automatic updates in the a server, it introduced a new vulnerability, in that time I din't have problem because I was using Snyk.